pkgs/development/libraries/sentencepiece/default.nix

```diff
@@ -0,0 +1,31 @@
+{ config
+, fetchFromGitHub
+, stdenv
+, lib
+, cmake
+, gperftools
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sentencepiece";
+  version = "0.1.84";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "144y25nj4rwxmgvzqbr7al9fjwh3539ssjswvzrx4gsgfk62lsm0";
+  };
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [ cmake gperftools ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/google/sentencepiece;
+    description = "Unsupervised text tokenizer for Neural Network-based text generation";
+    license = licenses.asl20;
+    platforms = platforms.linux;
```

Comment:
```suggestion
    platforms = [ "x86_64-linux" ];
```

the python package fails to build on all interpreter versions
```
cannot build derivation '/nix/store/8md1vqnkv98mm67q04xwwbgcr38aq6lp-python2.7-transformers-2.2.1.drv': 1 dependencies couldn't be built
/nix/store/m7lk7g5ccvg7x4h0jwigcb4lpxwzgk8a-setuptools-check-hook/nix-support/setup-hook: line 4:    75 Segmentation fault      (core dumped) /nix/store/9894fxjpg8v99j14kip6b13rdfgf4m5k-python3-3.7.5/bin/python3.7 nix_run_setup test
/nix/store/i3zis1yv4srhb62asn0442xjsf9ry0hl-setuptools-check-hook/nix-support/setup-hook: line 4:    75 Segmentation fault      (core dumped) /nix/store/hywqavwx22kk56lvsl06ihhswfr89dcp-python3-3.8.0/bin/python3.8 nix_run_setup test
builder for '/nix/store/q4m51ghk3y3n2ih0v2y5ls2y3xj19vs1-python3.7-sentencepiece-0.1.84.drv' failed with exit code 139
cannot build derivation '/nix/store/s50pg0qqxg8sqsrjdxvhkkpm8l0x5m4c-python3.7-transformers-2.2.1.drv': 1 dependencies couldn't be built
builder for '/nix/store/7bx6246na5r2s1jlwsy69siv7c1fw41d-python3.8-sentencepiece-0.1.84.drv' failed with exit code 139
cannot build derivation '/nix/store/xdhqracb4ib4dzqc9p7db50mp4xphv9y-python3.8-transformers-2.2.1.drv': 1 dependencies couldn't be built
error: build of '/nix/store/1iv6mnw2zlypi6fvnjzfq72fjv8gr685-python2.7-sentencepiece-0.1.84.drv', '/nix/store/7bx6246na5r2s1jlwsy69siv7c1fw41d-python3.8-sentencepiece-0.1.84.drv', '/nix/store/8md1vqnkv98mm67q04xwwbgcr38aq6lp-python2.7-transformers-2.2.1.drv', '/nix/store/q4m51ghk3y3n2ih0v2y5ls2y3xj19vs1-python3.7-sentencepiece-0.1.84.drv', '/nix/store/s50pg0qqxg8sqsrjdxvhkkpm8l0x5m4c-python3.7-transformers-2.2.1.drv', '/nix/store/xdhqracb4ib4dzqc9p7db50mp4xphv9y-python3.8-transformers-2.2.1.drv' failed
```

Reply:
done

Reply:
erm - think need to do for all parts as well - one sec

Reply:
@jonringer am i missing something with the checks? aarch64-linux seems to be failing for some reason...